reliquery/relic.py

```diff
@@ -252,13 +253,43 @@ def add_json(self, name: str, json_data: Dict) -> None:
     def list_json(self) -> List[str]:
         return self.storage.list_keys([self.relic_type, self.name, "json"])
 
-    def get_json(self, name: str) -> dict:
+    def get_json(self, name: str) -> Dict:
```

Comment:
One thing about typing. We can fix this later, but in general json can be a list, a dict, or a string or number. We can figure out the typing for this later as well.